etna/auto/pool/templates.py

```diff
@@ -0,0 +1,35 @@
+# fmt: off
+
+DEFAULT = [
+    {'_target_': 'etna.pipeline.Pipeline', 'horizon': '${__aux__.horizon}', 'model': {'_target_': 'etna.models.ProphetModel'}, 'transforms': [{'_target_': 'etna.transforms.TimeSeriesImputerTransform', 'in_column': 'target', 'strategy': 'forward_fill', 'window': -1}, {'_target_': 'etna.transforms.MedianOutliersTransform', 'in_column': 'target'}, {'_target_': 'etna.transforms.TimeSeriesImputerTransform', 'in_column': 'target', 'strategy': 'mean'}, {'_target_': 'etna.transforms.LinearTrendTransform', 'in_column': 'target'}]} ,
+    {'_target_': 'etna.pipeline.Pipeline', 'horizon': '${__aux__.horizon}', 'model': {'_target_': 'etna.models.HoltWintersModel', 'damped_trend': True, 'seasonal': 'add', 'trend': 'add'}, 'transforms': [{'_target_': 'etna.transforms.TimeSeriesImputerTransform', 'in_column': 'target', 'strategy': 'forward_fill'}, {'_target_': 'etna.transforms.HolidayTransform', 'out_column': 'holiday_regressor'}, {'_target_': 'etna.transforms.SegmentEncoderTransform'}]} ,
+    # {'_target_': 'etna.pipeline.Pipeline', 'horizon': '${__aux__.horizon}', 'model': {'_target_': 'etna.models.CatBoostMultiSegmentModel'}, 'transforms': [{'_target_': 'etna.transforms.TimeSeriesImputerTransform', 'in_column': 'target', 'strategy': 'mean'}, {'_target_': 'etna.transforms.LogTransform', 'in_column': 'target'}, {'_target_': 'etna.transforms.BinsegTrendTransform', 'in_column': 'target', 'jump': 1, 'min_size': 10}, {'_target_': 'etna.transforms.SegmentEncoderTransform'}, {'_target_': 'etna.transforms.LagTransform', 'in_column': 'target', 'lags': '${shift:${horizon},[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20]}'}, {'_target_': 'etna.transforms.DateFlagsTransform', 'day_number_in_month': True, 'day_number_in_week': True, 'is_weekend': True, 'month_number_in_year': True, 'week_number_in_month': True, 'week_number_in_year': True, 'year_number': True}]} ,
```

Comment:
What happened to these commented lines?

Reply:
I'am going to add extra task for add pipelines to templates